### ISSUE TITLE
[dev.icinga.com #3925] Icinga had forgotten to scheduling a servicecheck?

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga-core#1250]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/3925

**Created by Andrea on 2013-04-04 14:23:38 +00:00**

Assignee: _(none)_
Status: _Closed_  (closed on _2014-07-19 13:06:30 +00:00_)
Target Version: _(none)_
Last Update: _2014-07-19 13:06:30 +00:00 (in Redmine)_

    Icinga Version: 1.6.1-2
    OS Version: Ubuntu 12.04

---

One day, serveral hosts and services were down (ca. 07:45 clock). We had fixed the problem as quickly as possible (ca. 08:05 clock). We had then made an SLA-Report of the availability of our systems. Some hosts and services had a downtime of 16 hours. The other hosts and services, which had fallen out on the said day, had the right downtime (ca. 30 min.).

Surprisingly, all systems were shown green to the said day in the web interface. The 16 hours can´t be true.

So I analyzed the logfile of icinga with suprise!:

This is an excerpt from the log file of a server, whose downtime was allegedly 16 hours:

<code>
\[Tue Mar 19 07:57:57 2013\] HOST ALERT: prodserver;DOWN;SOFT;1;CRITICAL - Network Unreachable (10.66.77.0)

\[Tue Mar 19 07:58:27 2013\] SERVICE ALERT: prodserver;HTTP;CRITICAL;HARD;1;Network is unreachable

\[Tue Mar 19 07:58:57 2013\] HOST ALERT: prodserver;DOWN;SOFT;2;CRITICAL - Network Unreachable (10.66.77.0)
\[Tue Mar 19 08:00:07 2013\] HOST ALERT: prodserver;DOWN;SOFT;3;CRITICAL - Network Unreachable (10.66.77.0)
\[Tue Mar 19 08:01:17 2013\] HOST ALERT: prodserver;DOWN;SOFT;4;CRITICAL - Network Unreachable (10.66.77.0)
\[Tue Mar 19 08:02:27 2013\] HOST ALERT: prodserver;DOWN;SOFT;5;CRITICAL - Network Unreachable (10.66.77.0)
\[Tue Mar 19 08:03:37 2013\] HOST ALERT: prodserver;DOWN;SOFT;6;CRITICAL - Network Unreachable (10.66.77.0)
\[Tue Mar 19 08:04:47 2013\] HOST ALERT: prodserver;DOWN;SOFT;7;CRITICAL - Network Unreachable (10.66.77.0)
\[Tue Mar 19 08:05:57 2013\] HOST ALERT: prodserver;UP;SOFT;8;PING OK - Packet loss = 0%, RTA = 0.33 ms

\[Wed Mar 20 00:00:00 2013\] CURRENT HOST STATE: prodserver;UP;HARD;1;PING OK - Packet loss = 0%, RTA = 0.23 ms
\[Wed Mar 20 00:00:00 2013\] CURRENT SERVICE STATE: prodserver;HTTP;OK;HARD;1;HTTP OK: HTTP/1.1 200 OK - 1497 bytes in 0.029 second response time

</code>

=&gt; The SERVICE had after the "SERVICE HARD STATE" never a "SERVICE SOFT STATE". Icing had "forgotten" to scheduling a check for this service until 00:00 clock. By 0 clock icinga checks the status of all hosts and services regardless of their state.

The SLA-Report takes the difference between the first hard critical status and the first hard ok status of the service (therefore 16h :( ).

I can´t explain explain it, this bug never happend before ....

---

**Relations**:

* duplicates [#1782](https://dev.icinga.com/issues/1782)
